docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py

```diff
@@ -23,13 +25,15 @@ async def main():
         )
 
         # expose HTTP service to host
-        tunnel = client.host().tunnel(http_srv).start()
+        tunnel = await client.host().tunnel(http_srv).start()
 
         # get HTTP service address
-        tunnel.endpoint()
+        endpoint = await tunnel.endpoint()
 
         # access HTTP service from host
-        # commenting below as it's a blocking function, needs changes
+        r = requests.get("http://" + endpoint)
```

Comment:
requests is sync/blocking. [Use httpx](https://www.python-httpx.org/async/) instead:
```suggestion
        async with httpx.AsyncClient() as http:
            r = await http.get(f"https://{endpoint}")
```

Reply:
Updated!